com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkPrefab.cs

```diff
@@ -15,6 +22,25 @@ public class NetworkPrefab
         /// </summary>
         public GameObject Prefab;
 
+        public NetworkPrefabOverride Override;
+
+        /// <summary>
+        /// The original "source" prefab
+        /// </summary>
+        public GameObject OverridingSourcePrefab;
+
+        /// <summary>
+        /// The original "source" prefab's hash
+        /// This is used typically in multi-project patterns where a separate project contains the
+        /// source prefab and the GlobalObjectIdHash was copied and pasted into this field.
+        /// </summary>
+        public uint OverridingSourceHash;
```

Comment:
I would think you could derive this from OverridingSourcePrefab and wouldn't need to store it.  OR, actually, isn't this the hash that overrides the hash for OverridingSourcePrefab when NetworkPrefabOverride.Hash is enabled?  if so the comment consufed me

Reply:
Yeah... I think just changing the names around might make it clearer?
SourceHash
SourcePrefab
PrefabOverride